go.mod

```diff
@@ -81,38 +81,38 @@ require (
 	github.com/digitorus/pkcs7 v0.0.0-20230818184609-3a137a874352
 	github.com/distribution/reference v0.6.0
 	github.com/dustin/go-humanize v1.0.1
-	github.com/elastic/go-elasticsearch/v8 v8.13.1
+	github.com/elastic/go-elasticsearch/v8 v8.14.0
 	github.com/elimity-com/scim v0.0.0-20240320110924-172bf2aee9c8
 	github.com/evanphx/json-patch v5.9.0+incompatible
-	github.com/fsouza/fake-gcs-server v1.48.0
-	github.com/fxamacker/cbor/v2 v2.6.0
+	github.com/fsouza/fake-gcs-server v1.49.2
+	github.com/fxamacker/cbor/v2 v2.7.0
```

Comment:
This is probably harmless, although go-webauthn is still on 2.6.0. I'm inclined to let it be if it clears tests.